src/chi/components/popover/popover.scss

```
@@ -314,6 +314,99 @@
       }
     }
   }
+
+  &.-gradient {
+    border-radius: $popover-gradient-border-radius;
+
+    .chi-popover__header {
+      border-top-left-radius: $popover-gradient-border-radius;
+      border-top-right-radius: $popover-gradient-border-radius;
+      background: $popover-gradient-header-background;
+      color: $popover-gradient-header-color;
+      padding-bottom: $popover-gradient-header-padding-bottom;
+
+      & .chi-popover__title {
+        font-size: $font-size-xl;
+        font-weight: $font-weight-semi-bold;
+        line-height: $line-height-xl;
+      }
+    }
+
+    .chi-button {
+      &.-close {
+        top: $popover-gradient-close-button-top;
+
+        &,
+        .chi-icon {
+          color: $color-text-white;
+        }
+      }
+    }
+
+    // Position offsets for left/right start/end placements only
+    &[x-placement='right-start'],
+    &[x-placement='left-start'] {
+      top: -$popover-gradient-arrow-offset !important;
```

I see many `importants`.

```suggestion
      top: -$popover-gradient-arrow-offset !important;
```